components/profile-top-menu/profile-top-menu.view.html

```diff
@@ -0,0 +1,5 @@
+<span>
+	<i ng-if="session.id > -1 && !Video.user.data.pic" ng-click="switch();" class="material-icons">person_pin</i>
```

Comment:
where this pic is supposed to come from? if this actually work, it seems that will display current viewd video owner pic, and not logged in user pic, so I think it should be stored in session object once user backend model has pic field